api/v1alpha1/sparkcluster_webhook.go

```diff
@@ -0,0 +1,283 @@
+package v1alpha1
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/utils/pointer"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+const (
+	minSparkValidPort int32 = 1024
+	maxSparkValidPort int32 = 65535
+)
+
+var (
+	defaultSparkDashboardPort             int32 = 8265
+	defaultSparkClusterPort               int32 = 7077
+	defaultSparkEnableNetworkPolicy             = pointer.BoolPtr(true)
+	defaultSparkWorkerReplicas                  = pointer.Int32Ptr(1)
+	defaultSparkEnableDashboard                 = pointer.BoolPtr(true)
+	defaultSparkNetworkPolicyClientLabels       = map[string]string{
+		"spark-client": "true",
+	}
+	defaultSparkImage = &OCIImageDefinition{
+		Repository: "bitnami/spark",
+		Tag:        "3.0.2-debian-10-r0",
+	}
+)
```

Comment:
Same suggested format (`sparkDefault...` or `sparkMin...`) for these variables

Reply:
👍